PFR/ForMathlib/Entropy/RuzsaDist.lean

```diff
@@ -489,7 +496,7 @@ lemma condRuzsaDist_eq_sum {X : Ω → G} {Z : Ω → S} {Y : Ω' → G} {W : Ω
     all_goals {
       rw [Measure.map_apply ‹_›]
       convert measure_empty
-      simp [← FiniteRange.range]
+      simp only [← FiniteRange.range, Set.preimage_compl, Set.preimage_range, Set.compl_univ]
```

Comment:
Similarly, why this change?

Reply:
Again, I'm refactoring according to the heuristic "if a `simp` is non-finishing, then substitute it with `simp only [...]`"

Am I wrong? Are there other heuristics I can follow? Thanks.

Reply:
See https://github.com/teorth/pfr/pull/206#discussion_r1611105505